lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -91,7 +91,10 @@ protected static function get_blocks_metadata() {
 						break;
 					}
 
-					$element_selector[] = $selector . ' ' . $el_selector;
+					$el_selectors = explode( ',', $el_selector );
+					foreach ( $el_selectors as $el_selector_item ) {
+						$element_selector[] = $selector . ' ' . $el_selector_item;
+					}
```

Comment:
I encountered something similar. I think it's worth illustrating the sort of CSS rule you are handling here in a comment.